wave_vision/tests/fast_tests.cpp

```diff
@@ -0,0 +1,115 @@
+// Libwave Headers
+#include "wave/wave_test.hpp"
+#include "wave/vision/fast_detector.hpp"
+
+namespace wave {
+
+#define TEST_CONFIG "tests/config/fast.yaml"
+#define TEST_IMAGE "tests/data/lenna.png"
```

Comment:
No, keep Lenna! Its the de-facto example / test image everyone uses, and lets keep that tradition ;P (I'm pretty sure people don't know the significance of Lenna other than us two).